.github/config/testcases.json

```diff
@@ -4,35 +4,28 @@
 			"type": "EKS",
 			"targets": [
 				{
-					"name": "collector-ci-amd64-1-22",
-					"region": "us-west-2",
-					"excluded_tests": [
-						"containerinsights_eks_containerd"
-					]
-				},
-				{
-					"name": "collector-ci-amd64-1-23",
+					"name": "collector-ci-amd64-1-24",
```

Comment:
We should only be testing new clusters on the dev branch. Not clusters we are already testing against.

Reply:
Makes sense. I have updated the testcases